.flake8

```diff
@@ -1,2 +1,2 @@
 [flake8]
-exclude = build/*,venv/*,doc/source/*,force-bdss/*
+exclude = build/*,venv/*,doc/source/*,force-bdss/*,force-wfmanager/*
```

Comment:
I think the only exclude we need here is `doc/source/*`, the other folders are in different repos or don't exist here.
Assuming this works from the root folder of this repository anyway..

Reply:
Ah is this needed because we're cloning `force-bdss` and force-wfmanager` into this repo's root directory during our travis build?

Reply:
Yes, that's why.